src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -254,21 +254,21 @@ public bool WriteIndented
             }
         }
 
-        internal ClassMaterializer ClassMaterializerStrategy
+        internal MemberAccessor MemberAccessorStrategy
         {
             get
             {
-                if (_classMaterializerStrategy == null)
+                if (_memberAccessorStrategy == null)
                 {
 #if BUILDING_INBOX_LIBRARY
-                    _classMaterializerStrategy = new ReflectionEmitMaterializer();
+                    _memberAccessorStrategy = new ReflectionEmitMemberAccessor();
 #else
                     // todo: should we attempt to detect here, or at least have a #define like #SUPPORTS_IL_EMIT
-                    _classMaterializerStrategy = new ReflectionMaterializer();
+                    _memberAcceessorStrategy = new ReflectionMemberAccessor();
```

Comment:
Ah, missed the typo here. Will fix.

Reply:
Yes it's the reason why the netfx test is failing.

Reply:
Fixed.